DuckDuckGo.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -140,8 +140,8 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/apple/swift-argument-parser",
       "state" : {
-        "revision" : "46989693916f56d1186bd59ac15124caef896560",
-        "version" : "1.3.1"
+        "revision" : "c8ed701b513cf5177118a175d85fbbbcd707ab41",
+        "version" : "1.3.0"
```

Comment:
Feel like I've seen this a couple times lately. Wonder if one of our upstream dependencies isn't using a fixed version? Should probably reset this to the version on `main`.

Reply:
I'll update from main again before I merge :)

Reply:
Had to tell exclude to explicitly update it :/